docsite/rst/guide_docker.rst

```diff
@@ -23,10 +27,17 @@ Ansible offers the following modules for orchestrating Docker containers:
 
 
 Ansible 2.1.0 includes major updates to the Docker modules, marking the start of a project to create a complete and
-integrated set of tools for orchestrating containers. The docker_image and docker_login modules as well as the dynamic
-inventory script were refactored to use a common set of parameters and environment variables for shaping Docker API
-connections. The docker_container and docker_image_facts modules are new.
+integrated set of tools for orchestrating containers. In addition to the above modules, we are also working on the
+following:
+
+Still using Dockerfile to build images? Check out `ansible-container <https://github.com/ansible/ansible-container>`_,
+and start building images from your Ansible playbooks.
 
+Use the *shipit* command in `ansible-container <https://github.com/ansible/ansible-container>`_
```

Comment:
Use the double backticks as markup around shipit to make it stand out more and be consistent with other places in the Ansible docs:  " ``shipit`` "    (The double backticks are great when explaining variables, directory names, commands that don't need to be in code blocks, etc.)